src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/BLEU/RationalNumber.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP.BLEU;
```

Comment:
I think we may want to move the shared types like `NGram`, `RationalNumber` and `SmoothingFunction` to the root folder (and the NLP namespace) since they will likely also be used by other scores like GLEU.

Reply:
I renamed the utility classes to be in the Common namespace. I'm including only the important (i.e. public) types to be in the root namespace so it's easier to read.